binary_transparency/firmware/cmd/ft_monitor/main.go

```diff
@@ -113,6 +114,14 @@ func main() {
 				glog.Warningf("Firmware Statement decoding from manifest failed reason %q", err)
 				continue
 			}
+
+			// "Verify" the signature:
+			if ok, err := common.VerifySignature(stmt.Metadata, stmt.Signature); !ok {
+				glog.Warningf("Firmware signature verification failed reason %q", err)
```

Comment:
for consistency, the logged warning could be "... verification failed: %q", err

Reply:
ACK